### PR TITLE
update font awesome

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

**Comment by maelle** [comment]
I think it looks good? Places I've checked

* My author page :grin: 

* Packages page

* Footer (@jeroen although the flask is maybe a bit too fat now?)

File: themes/ropensci/layouts/partials/head.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <meta charset="utf-8" />
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/all.css">
```
Comment (maelle) [inline themes/ropensci/layouts/partials/head.html]: my issue in the previous PR was that I used min.css, all.css is needed to actually load the font... I've read it's a recent change.

**Comment by maelle** [comment]
Problem with the flask fixed, and I've added academicons to support ORCID.

**Comment by maelle** [comment]
@jeroen we'd switch back to font-awesome only when fa-orcid is added to font-awesome. :angel:

**Comment by jeroen** [comment]
> @jeroen we'd switch back to font-awesome only when fa-orcid is added to font-awesome. 👼

Can you add this as an HTML comment above the line so that we remember in future why this custom font is included?

**Comment by maelle** [comment]
Yes and I'll open an issue where I'm assigned. Other comments?

**Pull request merged**